buffer/src/main/java/io/netty/buffer/UnpooledUnsafeDirectByteBuf.java

```diff
@@ -224,21 +224,41 @@ protected short _getShort(int index) {
     }
 
     @Override
+    protected short _getShortLE(int index) {
+        return UnsafeByteBufUtil.getLEShort(addr(index));
```

Comment:
rename new methods in `UnsafeByteBufUtil` to be consistent with `ByteBuf` naming scheme.

`getLEShort` -> `getShortLE`.